enterprise/backend/src/metabase_enterprise/serialization/v2/seed_entity_ids.clj

```diff
@@ -0,0 +1,114 @@
+(ns metabase-enterprise.serialization.v2.seed-entity-ids
+  (:require
+   [clojure.string :as str]
+   [clojure.tools.logging :as log]
+   [metabase.db :as mdb]
+   [metabase.db.connection :as mdb.connection]
+   [metabase.models]
+   [metabase.models.dispatch :as models.dispatch]
+   [metabase.models.serialization.hash :as serdes.hash]
+   [metabase.util :as u]
+   [metabase.util.i18n :refer [trs]]
+   [toucan.db :as db]
+   [toucan.models :as models]))
+
+(set! *warn-on-reflection* true)
+
+;;; make sure all the models get loaded up so we can resolve them based on their table names.
+;;;
+;;; TODO -- what about enterprise models that have `entity_id`? Don't know of any yet. We'll have to cross that bridge
+;;; when we get there.
+(comment metabase.models/keep-me)
+
+(defn- entity-id-table-names
```

Comment:
Maybe dumb question - it seems more straightforward to query the models for the `entity_id` property, than to trawl the JDBC table metadata. Is there a reason why the more bedrock source is needed or preferred?

Reply:
The only reason I did it this way was because it seemed easier... I could do a single JDBC metadata query to get the info vs having to query each individual model